src/main/scala/org/allenai/pipeline/Workflow.scala

```diff
@@ -155,6 +155,11 @@ object Workflow {
     Workflow(nodes, links)
   }
 
+  def upstreamDependencies(step: PipelineStep): Set[PipelineStep] = {
```

Comment:
Let's make it clear these are the immediate deps. Or are they? Either way, let's clarify the intent :)